lib/everest.cpp

```diff
@@ -332,7 +332,7 @@ void Everest::publish_var(const std::string& impl_id, const std::string& var_nam
         } catch (const std::exception& e) {
             std::ostringstream oss;
             oss << "Publish var of " << this->config.printable_identifier(this->module_id, impl_id)
-                << " with variable name '" << var_name << "' with value '" << json_value
+                << " with variable name '" << var_name << "' with value '" << std::setw(4) << json_value
```

Comment:
You could also use
```cpp
json_value.dump(4)
```
here which [would indent the whole json properly.](https://json.nlohmann.me/api/basic_json/dump/)

Reply:
I think this is identical

Reply:
Thats true.